models/demos/llama3/tt/generator.py

```diff
@@ -368,14 +552,14 @@ def sample(logits):
         )
```

Comment:
This line `logits = logits.view(1, 1, self.model_args.vocab_size)` should be added in `generate()` after prefill_forward_single_user and before sample(logits) now that process_output_prefill returns 1d output